src/neoRpcConnection.ts

```diff
@@ -9,11 +9,24 @@ const BlocksPerPage = 20;
 const PollingInterval = 1000;
 
 export class BlockchainInfo {
+    public readonly populatedBlocksKnown: boolean;
+    public readonly populatedBlocks: BitSet;
     constructor(
         public height: number,
         public url: string,
         public online: boolean,
-        public chainIdentifier?: string) {
+        public chainIdentifier?: string,
+        populatedBlockList?: number[]) {
+
+        this.populatedBlocks = new BitSet();
+        if (populatedBlockList) {
+            this.populatedBlocksKnown = true;
+            for (let i = 0; i < populatedBlockList.length; i++) {
+                this.populatedBlocks.set(populatedBlockList[i]);
```

Comment:
Is there a more efficient JSON representation that express can return to improve how populatedBlocks is deserialized?

Reply:
you could give me a dictionary where the keys are block numbers (values don't matter) then I could just use that object directly (although it will be a bit more bloated than the internal representation of BitSet).  Something like { 1234: 1, 1235:1, 1237: 1, ... }

Reply:
What do we think the trade off is between larger payload side and BitSet deserialization time? Is this worth it?

Reply:
I'd guess (but haven't benchmarked) that if we're dealing with blockchains of that have <~ 1,000,000 populated blocks then it is probably best left as-is; the semantics of returning an array of ints from the RPC endpoint makes more logical sense and the parsing time in the tracker should be negligible.

If we do expect the massive chain scenario and want an easy improvement, then JSON object is the way to go.  It bakes a little bit of tracker implementation detail into Neo Express (having the values be keys in an object doesn't make much logical sense) but gets rid of all parsing code in the tracker.

The super-optimized implementation would be to return a string suitable for passing to the BitSet constructor (see: node_modules/bitset/bitset.js).  This bakes tracker (and BitSet) implementation details into Neo Express though, and will add a bit of complexity to Neo Express.

Reply:
lets leave it as is for now. If the bitset is written in JS, I'm not sure it makes much difference anyway